website/client/components/inventory/stable.vue

```diff
@@ -1,131 +1,195 @@
+      )
+        h4(v-once) {{ petGroup.label }}
+
+        .items
+          item(
+          v-for="pet in pets(petGroup, viewOptions[petGroup.key].open)",
+          :item="pet",
+          :key="pet.key",
+          :showPopover="true",
+          :label="pet.value",
+          :popoverPosition="'top'",
+          v-once
+          )
+            span(slot="popoverContent", v-once) {{ pet }}
+            div(slot="itemContent", :class="'Pet-' + pet.key", v-once)
+
+        .btn.btn-show-more(@click="viewOptions[petGroup.key].open = !viewOptions[petGroup.key].open") {{ viewOptions[petGroup.key].open ? 'Close' : 'Open' }}
+
+
+      h2 Mounts
+
+      h4 Standard Mounts
+      h4 Magic Potion Mounts
+      h4 Quest Mounts
+      h4 Special Mounts
 </template>
 
 <style lang="scss">
-.inventory-item-container {
-  padding: 20px;
-  border: 1px solid;
-  display: inline-block;
-}
+  .inventory-item-container {
+    padding: 20px;
+    border: 1px solid;
+    display: inline-block;
+  }
+
+  .item .item-content {
+    position: absolute;
+    top: -18px;
+    right: inherit;
+    display: block;
+  }
 </style>
 
 <script>
-import { mapState } from 'client/libs/store';
-import each from 'lodash/each';
-
-// TODO Normalize special pets and mounts
-// import Store from 'client/store';
-// import deepFreeze from 'client/libs/deepFreeze';
-// const specialMounts =
-
-export default {
-  data () {
-    return {
-      open: {
-        dropPets: false,
-        magicPets: false,
-        questPets: false,
-        rarePets: false,
-      },
-    };
-  },
-  computed: {
-    ...mapState(['content']),
-    dropPets () {
-      return this.listAnimals('pet', this.content.dropEggs, this.content.dropHatchingPotions, this.open.dropPets);
-    },
-    magicPets () {
-      return this.listAnimals('pet', this.content.dropEggs, this.content.premiumHatchingPotions, this.open.magicPets);
+  import {mapState} from 'client/libs/store';
+
+  import bDropdown from 'bootstrap-vue/lib/components/dropdown';
+  import bDropdownItem from 'bootstrap-vue/lib/components/dropdown-item';
+
+  import each from 'lodash/each';
+
+  import Item from 'client/components/inventory/item';
+  import Drawer from 'client/components/inventory/drawer';
+
+  // TODO Normalize special pets and mounts
+  // import Store from 'client/store';
+  // import deepFreeze from 'client/libs/deepFreeze';
+  // const specialMounts =
+
+  export default {
+    components: {
+      Item,
+      Drawer,
+      bDropdown,
+      bDropdownItem,
     },
-    questPets () {
-      return this.listAnimals('pet', this.content.questEggs, this.content.dropHatchingPotions, this.open.questPets);
+    data () {
+      return {
+        viewOptions: {},
+      };
     },
-    rarePets () {
-      return this.listAnimals('pet', this.content.dropEggs, this.content.dropHatchingPotions, this.open.rarePets);
+    computed: {
+      ...mapState(['content']),
+
+      petGroups () {
+        var petGroups = [
```

Comment:
nevermind I see it uses viewOptions. But consider this way the array is recalculated every time and the viewOptions set here will be reset too